pyproject.toml

```diff
@@ -137,22 +211,18 @@ docs = [
     "pymdown-extensions>=10.7",
 ]
 
-[tool.hatch]
-installer = "uv"
+[tool.hatch.env]
+requires = ["hatch-uvenv"]
+
 
 # Override the default uv version to use the latest version
 [tool.hatch.envs.hatch-uv]
 dependencies = ["uv>=0.4.17"]
 
 [tool.hatch.envs.default]
-features = ["dev"]
-python = "3.12"
 installer = "uv"
-dependencies = [
-    "uv>=0.4.17", # so we can bump the version used by hatch
-    # Typo checking
-    "typos~=1.23.6",
-]
+groups = [ "dev" ]
```

Comment:
There doesn't seem to be a whole lot of functional difference between sourcing from a `dependency-group` using the plugin vs putting them in the `tool.hatch` config, but tiebreaker goes to the standard